tests/features/table_delete.feature

```diff
@@ -0,0 +1,261 @@
+Feature: chadmin delete detached table commands
```

Comment:
Also lets add a test to cover cli options usage, suggest to use `Scenario Outline` for that.

Reply:
I think your suggestion is mostly relate with test coverage of existing functionality (table delete by pattern) than current adding feature of deleting particular table. So we have a lot of variants of using args/options, checking constraints when we use command table delete. I suppose it's better to do it in a separate task, if you dont mind.